<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CurrentLocationWidget</class>
 <widget class="QWidget" name="CurrentLocationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Current Location</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="projectionLabel_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Position Tracking</string>
     </property>
     <property name="buddy">
      <cstring>positionTrackingComboBox</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="positionTrackingComboBox">
     <item>
      <property name="text">
       <string>Disabled</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="showTrackCheckBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Show Track</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="clearTrackButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Clear Track</string>
       </property>
       <property name="icon">
        <iconset resource="../marble.qrc">
         <normaloff>:/icons/16x16/edit-clear-list.png</normaloff>:/icons/16x16/edit-clear-list.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="openTrackButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../marble.qrc">
         <normaloff>:/icons/16x16/document-open.png</normaloff>:/icons/16x16/document-open.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="saveTrackButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../marble.qrc">
         <normaloff>:/icons/16x16/document-save.png</normaloff>:/icons/16x16/document-save.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="locationLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>No position available.</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="recenterLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string comment="Select the method to use to recenter your GPS location on map.">Select the method to use to re-center your GPS location on map.</string>
     </property>
     <property name="text">
      <string>Map Adjustment</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="recenterComboBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string comment="Disable Auto Recenter">Disabled</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="Always keep gps location at the center of the map">Keep at Center</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="Auto center when required">When required</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="autoZoomCheckBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string comment="Auto Zoom to appropriate zoom level">Auto Zoom to appropriate zoom level</string>
     </property>
     <property name="text">
      <string>Auto Zoom</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../marble.qrc"/>
 </resources>
 <connections/>
</ui>
